ethereum/gateway-eth-ts/examples/eth-ts-calldata.js

```diff
@@ -15,7 +15,7 @@ require("dotenv/config");
     await gtCallDataLib.init();
     const testUser = '0x57AB42d4fa756b6956b0cAf986a5f53bA90D9e28';
     
-    let gatewayToken = await gtCallDataLib.gatewayTokens["0xa3894BbA27f4Be571fFA319D02c122E021024cF2"].tokenInstance
+    let gatewayToken = await gtCallDataLib.gatewayTokens["0x67306284Fb127E9baF713Ebf793d741cE763F81A"].tokenInstance
```

Comment:
Can you make this a constant in a shared file for all examples?

Reply:
Agreed. This should just come from the lib as well.

Reply:
This is an example file that is written with JS. The other occurrences on TS code were fixed.